Until we have eventfd, we need to link with epoll-shim.
See patch-Source_WebCore_SourcesGTK_txt

Index: Source/WebCore/CMakeLists.txt
--- Source/WebCore/CMakeLists.txt.orig
+++ Source/WebCore/CMakeLists.txt
@@ -1816,6 +1816,7 @@ set(WebCore_LIBRARIES
     LibXml2::LibXml2
     SQLite::SQLite3
     ZLIB::ZLIB
+    "epoll-shim"
 )
 set(WebCore_FRAMEWORKS
     JavaScriptCore
